solutions/community-deploy/cloud/AWS/setup.sh

```diff
@@ -21,27 +21,13 @@
 #Only root to should run setup
 if [ $(whoami) != root ];then
 	echo "$(whoami) is not root user. Please use root user to execute this script"
-	exit
 fi
 
-#Configure Elastic Network cards
-#echo "Configuring Elastic NIC"
-#for nic in eth1 eth2
-#do
-#   MAC=$(ifconfig $nic | grep -o -E '([[:xdigit:]]{2}:){5}[[:xdigit:]]{2}')
-#   cp /etc/sysconfig/network-scripts/ifcfg-eth0 /etc/sysconfig/network-scripts/ifcfg-$nic
-#   sed -i 's/eth0/$nic/g' /etc/sysconfig/network-scripts/ifcfg-$nic
-#   sed -i -e '/HWADDR/d' /etc/sysconfig/network-scripts/ifcfg-$nic
-#   sed -i "5 a HWADDR=$MAC" /etc/sysconfig/network-scripts/ifcfg-$nic
-#   echo -e "SUBSYSTEM==\"net\", ACTION==\"add\", DRIVERS==\"?*\", ATTR{address}==\"$MAC\", NAME=\"$nic\"" >> /etc/udev/rules.d/70-persistent-net.rules
-#done
-
-#echo "GATEWAYDEV=eth0" >> /etc/sysconfig/network
-#echo "Restarting Network Service"
-#systemctl restart network
-
-#echo "All networks should have ip address assigned" 
-#/sbin/ip a 
+#Install Docker engine on worker nodes
+echo -e "-------------------------[ Installing docker ]----------------------------------------" 
```

Comment:
We are installing `docker` as part of K8s installation script so do we need to do it again here?

Reply:
yes, as the K8s installation script is not installing on worker nodes so as an alternative & prerequisites added in the setup.sh itself

Reply:
We do install docker on the worker node as part of the K8s setup. Refer - https://github.com/Seagate/cortx-re/blob/main/solutions/kubernetes/cluster-functions.sh#L207

Reply:
I still see docker package is not installed on the worker nodes with the previous code as testing,

[root@ip-172-31-XX-YY tmp]# ls |grep tar
cortximages.tar
[root@ip-172-31-XX-YY tmp]# for image in /tmp/*.tar; do cat $image | docker load; done
bash: docker: command not found

[root@ip-172-31-XX-YY tmp]# ls |grep tar
cortximages.tar
[root@ip-172-31-XX-YY tmp]# for image in /tmp/*.tar; do cat $image | docker load; done
bash: docker: command not found

So, re-added the code to install docker